src/form/Form.tsx

```diff
@@ -1078,6 +1065,12 @@ function Form({
             (!actionConfig.isMatch ||
               (actionConfig.isMatch && actionConfig.isMatch(actionData)))
           ) {
+            // We only want to run firebase submit logic if it is happening via field auto-submit
+            if (
```

Comment:
can we fully remove auth-related logic from the handleActions path? this function is only called as part of the submission path. We've already been pulling auth logic out into a pre-submission path, and IMO we should double down on that